modin/pandas/series.py

```diff
@@ -1329,11 +1329,7 @@ def is_monotonic_increasing(df):
 
     @property
     def is_unique(self):
```

Comment:
@amyskov plz add docstring compatible with numpydoc standard (#1411)

Reply:
@anmyachev, numpy docstring is added.